.travis.yml

```diff
@@ -89,4 +90,19 @@ jobs:
         - $HOME/gopath/bin/goveralls -coverprofile=coverage.out -service=travis-ci
       if: type = pull_request OR branch = master
       after_success:
-        - coveralls
\ No newline at end of file
+        - coveralls
+
+    - stage: Analysis
+      name: Sonarcloud
+      if: type = pull_request
```

Comment:
I think this needs to be `type = pull_request OR branch = master` so that it updates the badge when merged into `master`